src/pages/iou/steps/NewRequestAmountPage.js

```diff
@@ -108,21 +114,27 @@ function NewRequestAmountPage({route, iou, report}) {
                 IOU.resetMoneyRequestInfo(moneyRequestID);
             }
 
-            if (_.isEmpty(iou.participants) || iou.amount === 0 || shouldReset) {
+            if (!isDistanceRequestTab && (_.isEmpty(iou.participants) || iou.amount === 0 || shouldReset)) {
                 Navigation.goBack(ROUTES.getMoneyRequestRoute(iouType, reportID), true);
             }
         }
 
         return () => {
             prevMoneyRequestID.current = iou.id;
         };
-    }, [iou.participants, iou.amount, iou.id, isEditing, iouType, reportID]);
+    }, [iou.participants, iou.amount, iou.id, isEditing, iouType, reportID, isDistanceRequestTab]);
 
     const navigateBack = () => {
         Navigation.goBack(isEditing ? ROUTES.getMoneyRequestConfirmationRoute(iouType, reportID) : null);
     };
 
     const navigateToCurrencySelectionPage = () => {
+        // If the money request being created is a distance request, don't allow the user to choose the currency.
+        // Only USD is allowed for distance requests.
```

Comment:
NAB: This comment is false, we can use other currencies, but this is not the way we support changing the currency right now. Maybe in the future we could actually update the default workspace currency from the currency selector for distance requests.

Idk there's a lot to consider there.

Reply:
that code is added as a result of the discussion [here](https://expensify.slack.com/archives/C05DWUDHVK7/p1693346905788559?thread_ts=1693332307.393409&cid=C05DWUDHVK7)

But I know you worked on applying the default currency of the workspace to the currency used in the distance request.

Let's leave the code as is (we expect all attendees of the conference to use USD), but I gonna definitely address this in a follow up